diff -ru Psopt3/PSOPT/src/psopt.cxx Psopt3-new/PSOPT/src/psopt.cxx
--- Psopt3/PSOPT/src/psopt.cxx	2014-01-15 14:55:35 +0000
+++ Psopt3-new/PSOPT/src/psopt.cxx	2014-01-15 15:00:50 +0000
@@ -7822,7 +7822,7 @@
 
          if (output!=NULL) fprintf(gscript,"\nset output \"%s\"", output);
 
-         fprintf(gscript,"\nset data style lines");
+         fprintf(gscript,"\nset style data lines");
 
          fprintf(gscript,"\nset multiplot");
 
@@ -7866,7 +7866,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 
 
@@ -7927,7 +7927,7 @@
 
          if (output!=NULL) fprintf(gscript,"\nset output \"%s\"", output);
 
-         fprintf(gscript,"\nset data style lines");
+         fprintf(gscript,"\nset style data lines");
 
          fprintf(gscript,"\nset multiplot layout %i,%i columnsfirst", nnrows,nncols);
 
@@ -8000,7 +8000,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 
 
@@ -8058,7 +8058,7 @@
 
          if (output!=NULL) fprintf(gscript,"\nset output \"%s\"", output);
 
-         fprintf(gscript,"\nset data style lines");
+         fprintf(gscript,"\nset style data lines");
 
          fprintf(gscript,"\nset multiplot");
 
@@ -8142,7 +8142,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 
 
@@ -8200,7 +8200,7 @@
 
          if (output!=NULL) fprintf(gscript,"\nset output \"%s\"", output);
 
-         fprintf(gscript,"\nset data style lines");
+         fprintf(gscript,"\nset style data lines");
 
          fprintf(gscript,"\nset multiplot");
 
@@ -8323,7 +8323,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 
 
@@ -8380,7 +8380,7 @@
 
          if (output!=NULL) fprintf(gscript,"\nset output \"%s\"", output);
 
-         fprintf(gscript,"\nset data style lines");
+         fprintf(gscript,"\nset style data lines");
 
          fprintf(gscript,"\nset multiplot");
 
@@ -8419,7 +8419,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 
 
@@ -8477,7 +8477,7 @@
 
          if (output!=NULL) fprintf(gscript,"\nset output \"%s\"", output);
 
-         fprintf(gscript,"\nset data style lines");
+         fprintf(gscript,"\nset style data lines");
 
          fprintf(gscript,"\nset multiplot");
 
@@ -8569,7 +8569,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 
 
@@ -8630,7 +8630,7 @@
 
          if (output!=NULL) fprintf(gscript,"\nset output \"%s\"", output);
 
-         fprintf(gscript,"\nset data style lines");
+         fprintf(gscript,"\nset style data lines");
 
          fprintf(gscript,"\nset multiplot");
 
@@ -8777,7 +8777,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 
 
@@ -8879,7 +8879,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 }
 
@@ -8973,7 +8973,7 @@
 
          fclose(gscript);
 
-         system("gnuplot -persist gnuplot.scp ");
+         system("gnuplot gnuplot.scp ");
 
 }
 
