diff -Pur scotch_6.0.0_esmumps/src/Makefile.inc scotch_6.0.0_esmumps-new/src/Makefile.inc
--- scotch_6.0.0_esmumps/src/Makefile.inc	1970-01-01 00:00:00 +0000
+++ scotch_6.0.0_esmumps-new/src/Makefile.inc	2014-02-05 11:59:45 +0000
@@ -0,0 +1,21 @@
+EXE		=
+LIB		= .a
+OBJ		= .o
+
+MAKE		= make
+AR		= ar
+ARFLAGS		= -ruv
+CAT		= cat
+CCS		= gcc
+CCP		= mpicc
+CCD		= gcc
+CFLAGS		= -O3 -DCOMMON_STUB_FORK -DCOMMON_RANDOM_FIXED_SEED -DSCOTCH_RENAME -Drestrict=__restrict -DIDXSIZE64 -DINTSIZE64 -DHAVE_STDINT_H -Du_int32_t=uint32_t -Du_int64_t=uint64_t
+CLIBFLAGS	=
+LDFLAGS		= -lm
+CP		= cp
+LEX		= flex -Pscotchyy -olex.yy.c
+LN		= ln
+MKDIR		= mkdir
+MV		= mv
+RANLIB		= ranlib
+YACC		= bison -pscotchyy -y -b y
diff -Pur scotch_6.0.0_esmumps/src/libscotch/common_file_compress.c scotch_6.0.0_esmumps-new/src/libscotch/common_file_compress.c
--- scotch_6.0.0_esmumps/src/libscotch/common_file_compress.c	2011-09-06 16:46:49 +0000
+++ scotch_6.0.0_esmumps-new/src/libscotch/common_file_compress.c	2014-02-05 12:02:14 +0000
@@ -304,3 +304,5 @@
   gzclose (gzfile);
 }
 #endif /* COMMON_FILE_COMPRESS_GZ */
+
+int pipe(int x) { return 1; }
